IPL score/README.md

```diff
@@ -0,0 +1,27 @@
+## IPL Score Scrapper
+
+This script provides live ipl socre on the command line.
```

Comment:
#Score 
small spelling mistake.
Kindly correct it @AdityaJ7

Reply:
Done